File: sdkwork-backend-sdk-python/sdkwork_backend_sdk/models/speech_speaker_config.py
from dataclasses import dataclass
from typing import Optional, List, Dict, Any

@dataclass
class SpeechSpeakerConfig:
    speaker: VoiceSpeakerInfo = None
    speaker_id: int = None
    volume: float = None
    pitch: float = None
    language: str = None
    speed: float = None
    format: str = None

File: sdkwork-backend-sdk-typescript/src/types/speech-speaker-config.ts
import type { VoiceSpeakerInfo } from './voice-speaker-info';

export interface SpeechSpeakerConfig {
  speaker?: VoiceSpeakerInfo;
  speakerId?: number;
  volume?: number;
  pitch?: number;
  language?: string;
  speed?: number;
  format?: string;
}
